packages/blocks/src/store/process-block-type.js

```diff
@@ -78,6 +78,7 @@ export const processBlockType =
 		const bootstrappedBlockType = select.getBootstrappedBlockType( name );
 
 		const blockType = {
+			apiVersion: 1,
```

Comment:
Like we see by needing to update a bunch of tests, this might break some tests or any custom extension logic that performs strict checking of block type. 

With that in mind, should we potentially document it as a breaking change in the package?

Reply:
This is more like internal breaking change, shouldn’t affect most of consumers. But I’ll add changelog entry.